index.html

```diff
@@ -43,4 +43,4 @@ <h3 class="tagline">A tale of <span class="tagline-1">passion</span> and <span c
     <script src="./src/Cover.js"></script>
     <script src="./src/main.js"></script>
   </body>
-</html>
```

Comment:
It looks like git thinks this is a "change", but the old and new code look identical to me. It's not something we necessarily need to fix in THIS commit, but in future commits, please look at the files changed and unstage any files like this where there isn't REALLY an intentional code change.